GeometryCore/src/Polygon.cpp

```diff
@@ -22,13 +23,26 @@ namespace geom_utils
     // Calculate area of polygon
     inline FPoint2D::coord Polygon::area() const 
     { 
-        return {}; 
+        if (this->points.size() < 3) return false; //can't bild a polygon from 2 points or less
```

Comment:
Polygon::area() should return FPoint2D::coord, not boolean.

Reply:
fixed